skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
```

Comment:
I agree that this function behaves differently than the threshold_* functions, but the name still feels too long! Do you really need the word 'apply' in there? Wouldn't "hysteresis_threshold()" work?! That being said, this is just a minor issue for me; I'm good with all the changes made.